forwards_paper.tex

```diff
@@ -361,9 +361,12 @@ \subsection*{Tables for succinct tree sequences}
     \end{center}
     \caption{
         An example tree sequence with three samples over a chromosome of length 10.
-        The left-hand panel shows the tree sequence pictorially in two different ways:
-        (top) as a sequence of tree topologies
-        and (bottom) the spatial extent of the edges that define these topologies.
+        The left-hand panels show the tree sequence pictorially in two different ways:
+        (top) a sequence of tree topologies; the first tree extends from genomic position 0 to 5,
+        and the second from 5 to 10; and
+        (bottom) the edges that define these topologies,
+        displayed over their corresponding genomic segment
+        (for instance, the edge from node 2 to node 4 is present only on the interval from 0 to 5).
```

Comment:
It'd be better to put the nodes in the lower-left panel above each other, so it didn't look like their x-positions meant something; but there's no room to do that, so I opted for more explanation here.

Reply:
The use of 'left-hand' and 'right-hand' panels is a bit ambiguous in a figure with 3 columns. Would it be easy to break it into two panels with A/B?